client/src/components/pages/ActivityInformation/index.js

```diff
@@ -1,36 +1,126 @@
 import React, { Component } from 'react';
+import axios from 'axios';
 import {StyledPage, StyledActivityDetailsNav} from "./index.style";
 import Header from "../../Header";
 import ActivityDetails from "../../ActivityDetails"
 import Table from '../../Table';
 
  class ActivityInformation extends Component {
 
-  constructor(props) {
-    super(props);
-  }
-
-  render() {
-    const { id } = this.props.match.params;
-    const columns = [
-      { Header:'id',accessor:'__id',show:false},
-      { Header: 'اسم الطالب',accessor:'name'},
-      { Header:'العمر',accessor:'age'},
-      { Header: 'الجنس',accessor:'gender'},
-    ];
-    return (
-        <React.Fragment>
-          <Header />
-          <StyledActivityDetailsNav>
-            <ActivityDetails activityId={id} />
-          </StyledActivityDetailsNav>
-          <StyledPage>
-            <Table columns={columns} data={[]}/>
-          </StyledPage>
-        </React.Fragment>
+   constructor(props) {
+     super(props);
+     this.state = {
+       suggestions: [],
+       userId: null,
+       userNameInput: '',
+     };
+   }
 
-     );
-    }
-  }
+   generateUserWithDate = (user) => {
+     return `${user.name} - ${user.dateOfBirth}`;
+   };
+
+   checkSuggestionsClick = (inputValue) => {
+     let checked = false;
+     this.state.suggestions.forEach((suggestion) => {
+       checked = checked ? checked : this.generateUserWithDate(suggestion) === inputValue;
```

Comment:
You don't need checked === checked here, it will automatically check if it's true. In this case it's false so write `!checked`

Reply:
I want the checked to be true on the end if anyone of the suggesstions gives true value.